download/download.go

```diff
@@ -19,13 +19,14 @@ import (
 // Usage text that will be displayed as command line help text when using the
 // `help download` command
 var Usage = `
-USAGE: %s download [url(s)]
-
-Download: Downloads files from the Sensitive Data Archive (SDA). If a directory is
-          provided (ending with "/"), then the tool will attempt to first
-          download the urls_list.txt file, and then download all files in this
-          list. If file urls are given, they files will be downloaded creating the
-		  same folder structure locally.
+USAGE: %s download (-outdir <dir>) [url(s) | file]
+
+Download: Downloads files from the Sensitive Data Archive (SDA). The files will
+		  be downloaded in the current directory, if outdir is not defined.
+		  If a directory is provided (ending with "/"), then the tool will attempt 
+		  to first download the urls_list.txt file, and then download all files in 
+		  this list. If file urls are given, they files will be downloaded creating 
+		  the same folder structure locally.
```

Comment:
```suggestion
Download: Downloads files from the Sensitive Data Archive (SDA). The files will
          be downloaded in the current directory, if outdir is not defined.
          If a directory is provided (ending with "/"), then the tool will attempt 
          to first download the urls_list.txt file, and then download all files in 
          this list. If file urls are given, they files will be downloaded creating 
          the same folder structure locally.
```

This is annoying, tabspace in a terminal is the equivalent of 8 spaces.
That means that the view in GH will never be aligned if it looks good in the console output. 😞

Reply:
Yes I just noticed that too. In VS it looks fine, but when checking it in the terminal it looks weird :/

Reply:
You can set the tabspace to 8 spaces in VS code

Reply:
Then it look fine in  both